mkosi/qemu.py

```diff
@@ -496,14 +497,14 @@ def start_journal_remote(config: Config, sockfd: int) -> Iterator[None]:
         with spawn(
             [
                 bin,
-                "--output", config.forward_journal,
+                "--output", joinpath("/work", str(config.forward_journal)),
```

Comment:
Why is this (and the rest in this file not using `workdir`?

Reply:
Good catch